### PR TITLE
Fixes #12 : Created social media section

### LINKED ISSUE #12: Add a social media section
Need to add Fossasia's social links onto the website.

Working on it!

### PR DESCRIPTION
**Pull request by saarthakchats**
Fixes #12 
- Created Social Media Section
- Added links to Facebook and Twitter with icons
- Linked YouTube
- Integrated a video that I made myself(hope you like it!)
[Preview Link](https://saarthakchats.github.io/ssasia.org/)

### CODE REVIEW

**Comment by saarthakchats** [comment]
@ms10398 @abishekvashok

File: index.html [hunk 1]
```diff
@@ -11,6 +11,7 @@
         <link href="css/ytplayer.css" rel="stylesheet" type="text/css" media="all" />
         <link href="css/theme.css" rel="stylesheet" type="text/css" media="all" />
         <link href="css/custom.css" rel="stylesheet" type="text/css" media="all" />
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
Comment (sk9331657) [inline index.html]: @saarthakchats  Don't use external libraries for just two icons. This makes site loading slower. Either save icon as images or use the already present sample icon for facebook and twitter.

Comment (saarthakchats) [reply index.html]: other participants can add icons for other social media also

Comment (saarthakchats) [reply index.html]: Thats why i left at 2 icons

**Comment by saarthakchats** [comment]
@sk9331657 please review and merge :)

**Comment by sarnava1** [comment]
LGTM but squash all your commits so that you have a single commit and then mentors will approve and merge it!

**Comment by saarthakchats** [comment]
It is an updated fork, and with that I have commits on top of it. So I am unable to squash as someone else's PR was merged before mine .  :(

**Comment by abishekvashok** [comment]
`git pull --rebase upstream`

**Pull request merged**